data/multiplayer/eras.cfg

```diff
@@ -3,7 +3,7 @@
 [era]
     id=era_default
     name= _ "Default"
-    description=_ "The standard era for Wesnoth multiplayer. Consists of six factions and is generally balanced."
+    description=_ "The standard era for Wesnoth multiplayer. Consists of seven factions and is generally balanced."
```

Comment:
Is the "generally balanced" statement still accurate?

Reply:
Yes.

Reply:
I'm curious what that is based on.

Reply:
The significant amount of balancing work that was done for them for 1.16 and 1.18.

Reply:
It is funny, at this point I firmly believe that Dunefolk are more well balanced than some default factions just because there was so much freedom when changing them. 

But yeah there is no concrete evidence, because there is no competetive environoment to check them in. So yeah watching / playing games is the best option right now.